Makefile

```diff
@@ -24,6 +24,11 @@ else
   Dockerfile_tag := '.''$(arch)'
 endif
 
+ifeq ($(arch), s390x)
+	race_flag := ''
```

Comment:
Why have you decided not to look for race conditions on s390x?

Reply:
race conditions are not supported on s390x.